src/utils.ts

```diff
@@ -0,0 +1,107 @@
+import { html, TemplateResult } from 'lit-html'
+import { Constants } from './constants'
+import { TSunCardI18N, TSunCardI18NKeys } from './types'
+
+export class I18N {
+  private localization: TSunCardI18NKeys
+  private dateFormatter: Intl.DateTimeFormat
+
+  constructor (language: string, use12HourClock: boolean) {
+    this.localization = Constants.LOCALIZATION_LANGUAGES[language]
+
+    // https://developer.mozill a.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Intl/DateTimeFormat/DateTimeFormat
```

Comment:
There's a space on `mozilla` that breaks the link

Reply:
Fixed